packages/edge/src/vercel-edge/utils/getAuthData.ts

```diff
@@ -0,0 +1,37 @@
+import { JWTPayload } from '@clerk/backend-core';
+import { GetSessionTokenOptions } from '@clerk/types';
+import { NextRequest } from 'next/server';
+
+import { ClerkAPI } from '../ClerkAPI';
+import { AuthData, WithAuthOptions } from '../types';
+
+export async function getAuthData(
+  req: NextRequest,
+  { sid, sub, loadSession, loadUser }: WithAuthOptions & JWTPayload,
+): Promise<AuthData> {
+  const getToken = (options: GetSessionTokenOptions = {}) => {
```

Comment:
❓ Are we going to update this to use the new getToken method we introduced in the nextjs/remix package?

Reply:
Yes, in a next PR as it was done for the other packages!

Reply:
Yes, in a next PR as it was done for the other packages!